14/umbraco-cms/fundamentals/backoffice/login.md

```diff
@@ -10,12 +10,67 @@ To access the backoffice, you will need to login. You can do this by adding `/um
 
 You will be presented with a login form similar to this:
 
-![Login screen](../../../../13/umbraco-cms/fundamentals/backoffice/images/login-backoffice-login.png)
+![Login screen](images/login-backoffice-login-v14.png)
 
 The **login** screen contains a short greeting, a **login form** and an optional **Forgotten password** link.
 
 Below, you will find instructions on how to customize the login screen.
 
+## Greeting
+
+The login screen features a greeting text: The "Welcome" headline. You can personalize this by overriding the existing language translation keys.
```

Comment:
```suggestion
The login screen features a greeting text: The "Welcome" headline. This can be personalized by overriding the existing language translation keys.
```